optuna/integration/allennlp.py

```diff
@@ -26,11 +26,13 @@ def dump_best_config(input_config_file: str, output_config_file: str, study: opt
 
     Args:
         input_config_file:
-            Input configuration file used with :class:`~optuna.integration.AllenNLPExecutor`.
+            Input configuration file (Jsonnet) specified in
+            :class:`~optuna.integration.AllenNLPExecutor`.
```

Comment:
I think the previous "used with" sounds better.

Reply:
OK, I reverted this change.